src/ai/backend/manager/api/scaling_group.py

```diff
@@ -31,13 +31,12 @@ class WSProxyVersionQueryParams:
 
 
 @aiotools.lru_cache(expire_after=30)  # expire after 30 seconds
-async def query_wsproxy_version(
+async def query_wsproxy_status(
     wsproxy_addr: str,
 ) -> str:
```

Comment:
The return type should be a `dict[str, Any]` as it seems to directly return the JSON object response.